server/server.js

```diff
@@ -2,12 +2,33 @@ const express = require('express')
 const app = express()
 const port = 3000
 
-const organizations = []
+const bodyParser = require('body-parser');
+app.use(bodyParser.urlencoded({ extended: true }));
+
+const organizations = {}
+let currId = 0

```

Comment:
Is there a benefit to doing using increment ids as apposed to using a uuid/guid?